### DISCUSSION TITLE
How to recognize zero digit in arabic  as PaddleOCR  completely ignore it

### DISCUSSION THREAD

**Discussion by abdelmageed95** [PaddlePaddle/PaddleOCR#12355] [Q&A]

**Comment by changdazhou** [comment]
It will not be ignored automatically. It may be that there are missing numbers in the model training data. Can you provide an example?

**Comment by abdelmageed95** [comment]
here are some examples :
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/83831812/da2dd8b0-aa56-4b65-bb53-9bb4a9dd3bde)

result:
'٢٧٨٤٦٣١١'
'٣/٤/٢٢٤١'
'٧/٨/٨٤٤١'

**Comment by Maryam-Mostafa** [comment]
Hello, I have the same issue. Is there any solution for this?

**Comment by MuhamedAdly** [comment]
I see the ar_dict.txt is missing the zero (٠) in Arabic, which proves it was missed while training the model.

https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppocr/utils/dict/ar_dict.txt
